### PR TITLE
missing label in LnurlWithdraw screen

### LINKED ISSUE #1274: Bug: LnurlWithdraw "Amount (Satoshi)" input is missing label
### Describe the bug

Inputs should always have a label if possible.
In this case there's a text which looks like a label but it's not `<label>`

### Screenshots

image

### To Reproduce

1. Visit to https://lnurl.fiatjaf.com/
1. Click on "lnurl-withdraw QR-code"
1. Alvy prompt opens
1. See `Amount (Satoshi)` isn't clickable.

### Expected behavior

`Amount (Satoshi)` is the label for the `input` below. They should be connected.

### Additional context

`dt`/`dd` are being used here. That's fine for normal text but not for a label/input combination.

### Are you working on this issue?

No

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Changes LnurlWithdraw screen to use TextField component

### Link this PR to an issue

Fixes #1274 

### Type of change (Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Tested Manually at https://lnurl.fiatjaf.com/

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/screens/LNURLWithdraw.tsx [hunk 1]
```diff
@@ -61,19 +61,26 @@ function LNURLWithdraw(props: Props) {
...
+        <ContentMessage
+          heading={`Amount (Satoshi)`}
+          content={`${minWithdrawable / 1000} sats`}
+        />
+      );
     } else {
       return (
-        <div className="mt-1 flex flex-col">
-          <Input
+        <dl className="my-4 p-4 shadow bg-white dark:bg-surface-02dp rounded-lg overflow-hidden">
```
Comment (escapedcat) [inline src/app/screens/LNURLWithdraw.tsx:72]: `dl` is not valid html with these child-components:
image
(plus some other issues)

Using a `div` here might solve the issue.

**Comment by escapedcat** [comment]
Would be nice to have tests that checks if the correct component is being rendered in each situation.

**Comment by im-adithya** [comment]
Done!

File: src/app/screens/LNURLWithdraw.tsx (hunk 1 above)
Comment (im-adithya) [reply src/app/screens/LNURLWithdraw.tsx]: Changed to div :+1:

**Review by escapedcat — APPROVED** [review]
Awesome!

File: src/app/screens/LNURLWithdraw/index.tsx [hunk 2]
```diff
@@ -61,11 +61,18 @@ function LNURLWithdraw(props: Props) {
...
+          heading={`Amount (Satoshi)`}
+          content={`${minWithdrawable / 1000} sats`}
+        />
+      );
     } else {
       return (
-        <div className="mt-1 flex flex-col">
-          <Input
+        <div className="my-4 p-4 shadow bg-white dark:bg-surface-02dp rounded-lg overflow-hidden">
+          <TextField
```
Comment (bumi) [inline src/app/screens/LNURLWithdraw/index.tsx:73]: should this even be a dual currency field? (that shows fiat amounts?)

Comment (escapedcat) [reply src/app/screens/LNURLWithdraw/index.tsx:73]: Good point. @im-adithya wanna give that a try?

**Review by lisabaut — APPROVED** [review]
Thanks

File: src/app/screens/LNURLWithdraw/index.tsx (hunk 2 above)
Comment (im-adithya) [reply src/app/screens/LNURLWithdraw/index.tsx:73]: Sure, initially thought of using that, will change now!

**Comment by bumi** [comment]
🚀

**Pull request merged**